skyvern/config.py

```diff
@@ -71,6 +71,7 @@ class Settings(BaseSettings):
     ENABLE_ANTHROPIC: bool = False
     ENABLE_AZURE: bool = False
     ENABLE_BEDROCK: bool = False
+    ENABLE_GPT_4O: bool = False
```

Comment:
You don't need these changes! The change is easier than you think :)

Reply:
Ahh,  i get it. Thanks for the feedback, tho. :)

Reply:
@Ghastly2001 are you open to updating the PR? I'd love to get this merged:)